lib/spandex.ex

```diff
@@ -358,24 +358,6 @@ defmodule Spandex do
     end
   end
 
-  @doc """
```

Comment:
it's already deprecated, I think because it's not good in most scenarios to continue the trace without knowing anything about the context other then the id. The priority for example is a super relevant thing and if someone want's to continue the trace without an explicit priority value taken from somewhere, they can. But it's good if they are forced to create the `SpanContext` struct so they stop and think about the context that they *might* be missing